app/views/govuk_publishing_components/components/docs/checkboxes.yml

```diff
@@ -164,6 +164,18 @@ examples:
           value: "green"
         - label: "Blue"
           value: "blue"
+  with_custom_heading_size:
+    data:
+      name: "favourite_colour"
+      heading: "What is your favourite colour?"
+      heading_size: "s"
```

Comment:
Perhaps this should be "small" rather than "s"?

Reply:
I do get it can be confusing but the classes are defined in govuk-frontend/Design System, so we have no control of that and I tried to avoid having an associated map based on which to generate these modifiers.